public/data/avie-dev.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Avie Fukai",
+  "type": "personal",
+  "bio": "A Filipino SE in Japan",
+  "avatar": "https://github.com/avie-dev.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/avie-dev",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AvieDev",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "www.linkedin.com/in/jfukai",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title":"Enters Japanese Language School in Chiba, Japan",
+      "date": "June 2010",
+      "icon": "graduation-hat",
+      "color": "pink",
+      "description": "Entered Asahi International School",
+      "url": ""
+    },
+    {
+      "title": "Started working as a Software Engineer in Tokyo",
+      "date": "April 2014",
+      "icon": "dollar",
+      "color": "green",
+      "description": "Started working in Japan",
+      "url": ""
```

Comment:
Please add your URL here or remove this line! `38`

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
@kumarsonsoff3 also removed this.